src/Components/PresentationalComponents/BaseDropdown/BaseDropdown.test.js

```diff
@@ -56,8 +55,10 @@ describe('BaseDropdown component', () => {
             </TestWrapper>
         );
 
-        userEvent.click(screen.getAllByTestId('dropdown-toggle')[0]); // open dropdown
-        await waitFor(() => userEvent.click(screen.getByTestId('dropdown-item1'))); // click first item
+        fireEvent.click(screen.getAllByTestId('dropdown-toggle')[0]); // open dropdown
```

Comment:
@gkarat All of my tests fail when I use `userEvent.click`.

Reply:
This is a good indication that something might be wrong with the UI because userEvent is the closest to real click simulation in a browser environment. Also, your tests can be failing because you don't use `await` before `userEvent.click` which is required. Are there any specific error messages?

Reply:
Anyway, feel free to ignore this right now. We can get back to this later on but please add // TODO note saying that we have to replace this eventually with `userEvent`.